<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>504</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>504</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>282</width>
    <height>504</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="UserListWidget" name="listFriends">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="buttonLogon"/>
   <addaction name="buttonReg"/>
   <addaction name="buttonAddFriend"/>
   <addaction name="buttonMChat"/>
   <addaction name="buttonLogoff"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="buttonLogon">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/png/res/ButtonLogon.png</normaloff>:/png/res/ButtonLogon.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>IDS_BUTTON_LOGON</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="buttonReg">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/png/res/ButtonReg.png</normaloff>:/png/res/ButtonReg.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>IDS_BUTTON_REG</string>
   </property>
  </action>
  <action name="buttonAddFriend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/png/res/ButtonAddFriend.png</normaloff>:/png/res/ButtonAddFriend.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>IDS_BUTTON_ADD_FRIEND</string>
   </property>
  </action>
  <action name="buttonMChat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/png/res/ButtonMChat.png</normaloff>:/png/res/ButtonMChat.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>IDS_BUTTON_MCHAT</string>
   </property>
  </action>
  <action name="buttonLogoff">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/png/res/ButtonLogoff.png</normaloff>:/png/res/ButtonLogoff.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>IDS_BUTTON_LOGOFF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UserListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">UserListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
